api/lib/notification_parser.js

```diff
@@ -87,6 +87,14 @@ NotificationParser.prototype.parse = function(notification_details) {
       previous_transaction_identifier;
   }
 
+  // Add urlBase to each url in notification
+  Object.keys(notification).forEach(function(key) {
```

Comment:
This is a little fragile: if someone adds a new key containing "url" it will modify it. Could you just add `urlBase` in the initial assignment above?